src/v0/destinations/tiktok_ads/data/TikTokTrack.json

```diff
@@ -19,6 +19,31 @@
     "sourceKeys": "properties.contents",
     "required": false
   },
+  {
```

Comment:
Do we expect all these sourceKeys coming at root level ? All these seems to be coming under `properties.contents.` or `properties.products.`

Reply:
There are not test cases with this `properties` structure. Is it to capture any corner scenarios ?

Reply:
Tiktok expects these attributes inside properties object. And we also send them in the root of properties according to our e-commerce spec. I will add one test case covering this.